charts/gateway-operator/values.yaml

```diff
@@ -0,0 +1,18 @@
+image:
+  repository: docker.io/kong/gateway-operator-oss
```

Comment:
We should default to the `docker.io.kong/gateway-operator` (enterprise) image.

Reply:
Yes. It has already been addressed in this commit. https://github.com/Kong/charts/pull/1023/commits/52bbfb9d451ba80b910bb209614631b6a0947161